test/unit/testExecutorNumber.js

```diff
@@ -1,11 +1,84 @@
 'use strict';
 const initSteps = require('app/core/initSteps');
 const {expect, assert} = require('chai');
+const steps = initSteps([`${__dirname}/../../app/steps/action/`, `${__dirname}/../../app/steps/ui`]);
+const ExecutorsNumber = steps.ExecutorsNumber;
 
-describe('ExecutorNumber', () => {
+describe('ExecutorsNumber', () => {
     let ctx;
     let formdata;
-    const ExecNumber = initSteps([`${__dirname}/../../app/steps/action/`, `${__dirname}/../../app/steps/ui`]).ExecutorsNumber;
+
+    describe('getUrl()', () => {
+        it('should return the correct url', (done) => {
+            const url = ExecutorsNumber.constructor.getUrl();
+            expect(url).to.equal('/executors-number');
+            done();
+        });
+    });
+
+    describe('handlePost()', () => {
+        let ctx;
+        let errors;
+        let formdata;
+        let session;
+        let hostname;
+        let featureToggles;
+
+        it('should return the ctx with the deceased married status and the screening_question feature toggle', (done) => {
+            ctx = {
+                executorsNumber: '3'
+            };
+            errors = {};
+            [ctx, errors] = ExecutorsNumber.handlePost(ctx, errors, formdata, session, hostname, featureToggles);
+            expect(ctx).to.deep.equal({
+                executorsNumber: '3',
+                isToggleEnabled: false
+            });
+            done();
+        });
+    });
+
+    describe('nextStepOptions()', () => {
+        it('should return the correct options when the FT is off', (done) => {
+            const ctx = {
+                isToggleEnabled: false
+            };
+            const nextStepOptions = ExecutorsNumber.nextStepOptions(ctx);
```

Comment:
This test shouldn't really require any refactoring to account for the change suggested above since the logic is the same.